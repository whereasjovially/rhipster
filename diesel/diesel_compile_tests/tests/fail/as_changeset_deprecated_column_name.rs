#[macro_use]
extern crate diesel;

table! {
    users {
        id -> Integer,
        name -> Text,
    }
}

#[derive(AsChangeset)]
#[table_name(users)]
struct UserForm {
    id: i32,
    #[column_name(name)]
    name: String,
}

fn main() {
    // Workaround for https://github.com/dtolnay/trybuild/issues/8
    compile_error!("warnings");
}
