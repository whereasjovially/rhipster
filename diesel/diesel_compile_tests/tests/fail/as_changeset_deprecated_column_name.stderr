warning: The form `table_name(value)` is deprecated. Use `table_name = "value"` instead
  --> $DIR/as_changeset_deprecated_column_name.rs:12:3
   |
12 | #[table_name(users)]
   |   ^^^^^^^^^^^^^^^^^

warning: The form `column_name(value)` is deprecated. Use `column_name = "value"` instead
  --> $DIR/as_changeset_deprecated_column_name.rs:15:7
   |
15 |     #[column_name(name)]
   |       ^^^^^^^^^^^^^^^^^

error: warnings
  --> $DIR/as_changeset_deprecated_column_name.rs:21:5
   |
21 |     compile_error!("warnings");
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^
